configFile.txt

```diff
@@ -28,30 +28,32 @@
 'numPedEvents'          : -1,
 'pedExclRegion'         : None,
 'rebin'                 : 4,
-'nsigma'                : 1.2,
+'nsigma'                : 1.8,
```

Comment:
I think we should really make two separate files if many of these parameters change

Reply:
Sì hai ragione penso sia più comodo avere i diversi configfile e associarci i pedestal. Anche meno if clauses così, cambio e aggiorno.